tests/update/updates_packagekit_gpk.pm

```diff
@@ -14,13 +14,23 @@ use base "x11test";
 use strict;
 use testapi;
 use utils;
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_leap);
 
 sub turn_off_screensaver {
+    my ($self) = @_;
     # Turn off screensaver
     x11_start_program('xterm');
     become_root;
     ensure_serialdev_permissions;
+
+    # Trying to detect bsc#1052258 which leads to the test failure occasionally on Leap 42.3
+    if (is_leap('=42.3') && script_run("dmesg | grep -P 'soft lockup - CPU#\\d+ stuck'")) {
+        # Fail on cpu stuck, see bsc#1052258 and collect logs if machine still responds
+        record_soft_failure 'cpu soft lockup detected, see bsc#1052258';
```

Comment:
I assume you have already seen this problem in recent tests. But the last comment in the bug was from 2017-10-29 and stating that the problem is not there anymore so please comment on the bug or explain what this PR has to do with the bug then

Reply:
but the bug is in state NEW , probably developer not sure enough to close it ?
I think it is a good catcher for this bug.
@rwx788 +1

Reply:
Bug is in the version kernel which we have in leap 42.3, which we use to verify upgrade scenarios. With other kernel versions it's not there. I'll put this in the code comment too.